modules/vcf2maf/1.0/vcf2maf.smk

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env snakemake
+
+
+##### ATTRIBUTION #####
+
+
+# Original Author:  Bruno Grande
+# Module Author:    Helena Winata
+# Contributors:     N/A
+
+
+##### SETUP #####
+
+import sys, os
+from os.path import join
+
+import oncopipe as op
+
+# Setup module and store module-specific configuration in `CONFIG`
+CONFIG = config["lcr-modules"]["vcf2maf"]
+LOG = "/logs/" + op._session.launched_fmt
+
+wildcard_constraints:
```

Comment:
Why is this needed? I haven't come across this in other modules

Reply:
it's because we're not using the `op.setup_module` function. That's because this module is meant to run as an add on to variant callers. So it doesn't have the typical `00-inputs` and `99-outputs` directory. It will simply be generated in a `##-maf` directory within strelka/manta/varscan and be symlinked into their respective `99-outputs/maf/`